main.py

```diff
@@ -271,8 +273,12 @@
                  'value': 'jotc'
              },
              {'label': 'Judgment of Wisdom', 'value': 'jow'},
+             {'label': 'Misery / Improved Faerie Fire', 'value': 'misery'},
+             {'label': 'Shadow Mastery / Improved Scorch / Winter\'s Chill',
+              'value': 'shadow_mastery'
```

Comment:
same suggestion
```suggestion
             {
                 'label': 'Shadow Mastery / Improved Scorch / Winter\'s Chill',
                 'value': 'shadow_mastery'
```

Reply:
okok I will change it right now, btw, do you know where i can find info for spell resistance?